### DISCUSSION TITLE
so-steno fails to start on standalone installation

### DISCUSSION THREAD

**Discussion by James-AAL** [Security-Onion-Solutions/securityonion#11574] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

12

### RAM

128gb

### Storage for /

12T

### Storage for /nsm

system-nsm  252:2    0 11.9T  0 lvm  /nsm

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

so-steno fails to start. so-status reports it as missing.

The error log shows the following:

2023-10-19T10:29:02.699068Z T:4f4497 [stenotype.cc:567] Starting, page size is 4096
2023-10-19T10:29:02.699158Z T:4f4497 [stenotype.cc:594] Setting up AF_PACKET sockets for packet reading
2023-10-19T10:29:03.548432Z T:4f4497 [stenotype.cc:262] Dropping privileges
2023-10-19T10:29:03.548467Z T:4f4497 [stenotype.cc:267] Dropping priviledges from 941 to GID stenographer
2023-10-19T10:29:03.548745Z T:4f4497 [stenotype.cc:279] Dropping priviledges from 941 to UID stenographer
2023-10-19T10:29:03.549793Z T:ccf707 [stenotype.cc:466] Thread 0 starting to process packets
2023-10-19T10:29:03.550024Z T:ccf707 [aio.cc:190] Opening packet file /tmp/stenographer195671115/PKT0/.1697711343549889: 4
2023-10-19T10:29:03.738097Z T:ccf707 [stenotype.cc:537] Thread 0 stats: MB=1 secs=5e-06 MBps=200000 packets=1763 blocks=1 polls=1 drops=0 drop%=0

**_panic: runtime error: slice bounds out of range [:2270554] with capacity 256_**

goroutine 7 [running]:
github.com/golang/leveldb/table.(*blockIter).Next(0xc00162e1e0, 0xc00068c600)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/golang/leveldb/table/reader.go:124 +0x415
github.com/golang/leveldb/table.block.seek(0xc001634000, 0xbbd, 0xbc2, 0xbd6d20, 0x16fc060, 0xc0004ce000, 0x1, 0x1, 0x10, 0x0, ...)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/golang/leveldb/table/reader.go:85 +0x190
github.com/golang/leveldb/table.(*Reader).find(0xc001632000, 0xc0004ce000, 0x1, 0x1, 0x0, 0x0, 0x16fc060, 0xc000122b00)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/golang/leveldb/table/reader.go:369 +0xef
github.com/golang/leveldb/table.(*Reader).Get(0xc001632000, 0xc0004ce000, 0x1, 0x1, 0x0, 0x1, 0x100000000000003, 0x0, 0x30, 0xc001df7200)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/golang/leveldb/table/reader.go:337 +0x8d
github.com/google/stenographer/indexfile.NewIndexFile(0xc001df7200, 0x30, 0xc0001edc80, 0x3, 0xaff380, 0x3)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/google/stenographer/indexfile/indexfile.go:67 +0x23d
github.com/google/stenographer/blockfile.NewBlockFile(0xc001df6600, 0x30, 0xc0001edc80, 0xc001df6600, 0x30, 0x1)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/google/stenographer/blockfile/blockfile.go:63 +0x14b
github.com/google/stenographer/thread.(*Thread).trackNewFile(0xc0001e7d60, 0xc000c886b0, 0x10, 0x10, 0x16fe000)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/google/stenographer/thread/thread.go:172 +0xc9
github.com/google/stenographer/thread.(*Thread).syncFilesWithDisk(0xc0001e7d60)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/google/stenographer/thread/thread.go:139 +0x2af
github.com/google/stenographer/thread.(*Thread).SyncFiles(0xc0001e7d60)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/google/stenographer/thread/thread.go:363 +0x45
github.com/google/stenographer/env.(*Env).syncFiles(0xc0001a00e0)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/google/stenographer/env/env.go:267 +0x50
github.com/google/stenographer/env.(*Env).callEvery(0xc0001a00e0, 0xc0002786d0, 0x37e11d600)
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/google/stenographer/env/env.go:180 +0x83
created by github.com/google/stenographer/env.New
        /home/mreeves/rpmbuild/BUILD/stenographer-e8db1ee8f6b62790ebc8081e464dcedd7c961766/src/github.com/google/stenographer/env/env.go:131 +0x3f8
2023-10-19T10:29:06.157915Z T:cd7717 [stenotype.cc:437] Got signal 15, stopping threads
2023-10-19T10:29:06.158191Z T:ccf707 [aio.cc:122] Closing /tmp/stenographer195671115/PKT0/.1697711343549889 (4), truncating to 7MB and moving to /tmp/stenographer195671115/PKT0/1697711343549889
2023-10-19T10:29:06.158283Z T:ccf707 [stenotype.cc:557] Finished thread 0 successfully

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by cm-ops** [comment]
Try moving the `/nsm/pcap` and `/nsm/pcapindex` directories to `.old` extensions and run a highstate.
